src/modules/manaFiatGateway/selectors.ts

```diff
@@ -8,9 +9,15 @@ import { ManaFiatGatewayState } from './reducer'
 export const getState: (state: any) => ManaFiatGatewayState = state =>
   state.manaFiatGateway
 
+export const getData = (state: any) => getState(state).data
 export const getLoading = (state: any) => getState(state).loading
 export const getError = (state: any) => getState(state).error
 
+export const getPendingPurchase = (state: any) =>
+  getData(state).purchases?.find(
```

Comment:
```suggestion
  getData(state).purchases.find(
```
I don't think the `?` is needed here since it's already initialized as an empty array, it should never be `undefined`